.travis.yml

```diff
@@ -3,8 +3,20 @@ go:
     - 1.7
     - 1.8
     - tip
-script: go test -v -randtests 1000 ./...
+before_script:
+    - PKGS=$(go list ./... | grep -v /vendor/)
+    - GO_FILES=$(find . -iname '*.go' -not \( -path '*/vendor/*' -prune \))
+    - go get -u honnef.co/go/tools/cmd/gosimple
```

Comment:
Can you merge these `go get` lines into one, like: `go get honnef.co/go/tools/cmd/{gosimple,staticcheck,unused}`

Also can you add the -v flag for consistency with what travis [already does](https://travis-ci.org/tmthrgd/go-bindata/jobs/220826486#L166).

You can also remove the -u flag because it's redundant as travis starts with a fresh GOPATH for each build, it's just clutter.

So replacing these three lines with: `go get -v honnef.co/go/tools/cmd/{gosimple,staticcheck,unused}`